README.md

```diff
@@ -156,6 +156,10 @@ defined, this parameter **must** be an integer!
 
   Optional parameter that controls how many seconds django-stomp will wait for a message to be processed. Defaults to 30 seconds. If defined, this parameter **must** be an integer!
 
+***STOMP_DEFAULT_EXCLUSIVE_QUEUE***
+
+  Optional parameter that defines if the default value of `exclusive` queue parameter will be True or False in RabbitMQ. The default is `False`.
```

Comment:
[suggestion]
We can be more explicit when this is applied:

```suggestion
  Optional parameter that defines the default value of `exclusive` queue parameter will be True or False when creating a queue in RabbitMQ. The default is `False`.
```

Reply:
Good point! I'll fix it!